README.md

```diff
@@ -2,62 +2,51 @@
 
 # End to end tests for Archive.org using [Playwright](https://playwright.dev/)
 
+
 ## (Optional) BrowserStack Local Setup
 
-- login to BrowserStack and retrieve the account username and access key, see documentation [here](https://www.browserstack.com/docs/automate/playwright/getting-started/nodejs/test-runner)
+- Login to BrowserStack and retrieve the account username and access key, see documentation [here](https://www.browserstack.com/docs/automate/playwright/getting-started/nodejs/test-runner)
 
 
-## Running tests locally
+## Local Setup
 
-- install dependencies:
+- Install dependencies:
 
     `npm i`
 
-- install Playwright browser libs:
+- Install Playwright browser libs:
 
     `npx playwright install`
 
-- run all the tests in headless mode and generate 1 whole test report:
-
-    `npm run test`
+- Create a `.env` file by copying the contents from `.env.sample` and add the respective values you want to use for testing
 
-- run all the tests in headless mode by each category and generate test report by category:
+    - this is required to run tests with loggedIn flows
 
-    `./run-tests.sh`
 
-- run all the tests in headed mode (this will load multiple browsers):
+## Running tests
 
-    `npm run test:headed`
+- Run all tests: `npm run test`
 
-- create a `.env` file by copying the contents from `.env.sample` and add the respective values you want to use for testing
+- Run test by category (sample):
 
+    - This will run a custom script that can take different shell arguments/parameters for the following (arguments arrangement can be jumbled or not in proper order):
+        
+        - test category (by folder structure): 
+            - `about, av, books, collection, details, donation, etc...`
+            - default value: `all`
 
-## Running individual tests by category (headless mode)
+        - test execution mode: 
+            - `headless/headed`
+            - default value: `headless`
 
-- run about tests: `npm run test:about`
-- run av tests: `npm run test:av`
-- run books tests: `npm run test:books`
-- run collection tests: `npm run test:collection`
-- run details tests: `npm run test:details`
-- run home tests: `npm run test:home`
-- run login tests: `npm run test:login`
-- run music tests: `npm run test:music`
-- run search tests: `npm run test:search`
-- run profile tests: `npm run test:profile`
+        - browser: 
+            - `chromium, firefox, webkit`
+            - default value: `chromium`
```

Comment:
Suggestion: prefer concise labels
`- default:`

Question: shouldn't the default be `all`?

Reply:
yes, it is all browsers by default... just got it wrong in the README